templates/snapcraft.yaml.j2

```diff
@@ -16,13 +16,37 @@ name: {{ snapname }}
 version: '0.1'
 summary: The ROS {{ versionmap[ros_distro] }} {{ ros_distro }} {{ variant }} variant.
 description: |
-  This snap contains the ROS {{ ros_distro }} {{ variant }} variant [1].
-  It is meant to be consumed as a stage or build snap.
+  This snap contains the ROS {{ versionmap[ros_distro] }} {{ ros_distro }} {{ variant }} variant [1].
+
+  It provides the ROS {{ versionmap[ros_distro] }} {{ ros_distro }} stack to other snaps to that use it.
+  It share the ROS {{ versionmap[ros_distro] }} {{ ros_distro }} libraries, components and executables
```

Comment:
Should we had that this snap is not meant to be used standalone?

Reply:
I think this is already explicited, especially in the 'for users' section.